File: DSA basic q for beg/checkprimeno.py
def checkprimeno(n):

    if n % 2 == 0:
        print("Number is not prime")
    elif n ==2:
        print("number is prime")        

    elif n<=1:
        print("Number is not prime")
    else:
        for i in range(3,int(n**0.5)+1,2):
            if n % i == 0:
                print(n, " : Number is not prime")
                return False
        print(n ,": Number is prime")
        return True       
    
n = 49
checkprimeno(n)

File: DSA basic q for beg/java dsa beginers/Checkprimeno.java
class Checkprimeno {

    public String checkPrime(int n) {
        if (n % 2 == 0) {
            System.out.println("not a prime number");
        } else if (n < 1) {
            System.out.println("not a prime number");
        } else if (n == 2) {
            System.out.println(" a prime number");

        } else {
            for (int i = 3; i <= Math.sqrt(n); i += 2) {
                if (n % i == 0) {
                    System.out.println("not a prime no");

                }

            }
        }
        return null;
    }

    public static void main(String[] args) {
        Checkprimeno chck = new Checkprimeno();
        int n = 12;
        String answer = chck.checkPrime(n);
        System.out.println("Result: " + answer);

    }
}
